tests/SprykerTest/Shared/Propel/_support/Helper/PropelBuildModelHelper.php

```diff
@@ -0,0 +1,254 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerTest\Shared\Propel\Helper;
+
+use Codeception\Module;
+use Spryker\Zed\Propel\Business\Model\PropelGroupedSchemaFinder;
+use Spryker\Zed\Propel\Business\Model\PropelGroupedSchemaFinderInterface;
+use Spryker\Zed\Propel\Business\Model\PropelSchema;
+use Spryker\Zed\Propel\Business\Model\PropelSchemaFinder;
+use Spryker\Zed\Propel\Business\Model\PropelSchemaFinderInterface;
+use Spryker\Zed\Propel\Business\Model\PropelSchemaInterface;
+use Spryker\Zed\Propel\Business\PropelFacade;
+use Spryker\Zed\Propel\Communication\Console\BuildModelConsole;
+use SprykerTest\Shared\Testify\Helper\ConfigHelperTrait;
+use SprykerTest\Zed\Testify\Helper\Business\BusinessHelperTrait;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\ConsoleOutput;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+
+class PropelBuildModelHelper extends Module
+{
+    use ConfigHelperTrait, BusinessHelperTrait;
+
+    /**
+     * @var string
+     */
+    protected const CONFIG_SCHEMA_SOURCE_DIRECTORY_LIST = 'schemaSourceDirectoryList';
+
+    /**
+     * @var string
+     */
+    protected const CONFIG_SCHEMA_TARGET_DIRECTORY = 'schemaTargetDirectory';
+
+    /**
+     * @var string
+     */
+    protected const CONFIG_IS_ISOLATED_MODULE_TEST = 'isolated';
+
+    /**
+     * @var string
+     */
+    protected const PROPEL_MODULE_NAME = 'Propel';
+
+    /**
+     * @var string
+     */
+    protected const PROPEL_MODULE_NAMESPACE = 'Spryker\Zed\Propel';
+
+    /**
+     * @var string
+     */
+    protected const PROPEL_MODULE_NAMESPACE_BUSINESS = self::PROPEL_MODULE_NAMESPACE . '\Business';
+
+    /**
+     * @var string
+     */
+    protected const SCHEMA_FILE_PATTERN = '*.schema.xml';
+
+    /**
+     * @var string
+     */
+    protected const SCHEMA_TARGET_DIRECTORY_DEFAULT = '/src/Orm/Propel/Schema';
```

Comment:
Why does this have a leading slash, but the one below does not?
Also, both should have a trailing slash as for all our paths IMO.